docs/src/operations/validator-initiatives.md

```diff
@@ -13,12 +13,15 @@ The Solana Foundation helps facilitate the growth of the consensus validator net
 
 ## Tour De Sun 22
 
-Delegation program participants who operate a performant testnet node, may also join Tour De Sun 22 (TdS22).  The program gives locked SOL incentive payments to operators for running approved hardware on Testnet. For more information, see [TdS22](https://solana.org/tds22).
+Tour De Sun 22 is winding down. Applications to the program are now closed.
+March 2024 was the final month of the program. The program is closed and will not accept new applicants.
+
+Please see the [End of Tour De Sun 22 blog post](https://solana.com/news/solana-foundation-announces-end-of-tour-de-sun-22) for more information.
 
 ## Solana Foundation Server Program
 
 Separately from the delegation program, The Solana Foundation offers a server program that provides servers in various data-centers all over the world. If you would like to run a consensus validator or RPC node, you may use this program to rent bare metal servers in various data-centers. The servers meet or exceed the [Solana validator hardware specs](./requirements.md#hardware-recommendations). No long-term lease commitments are required. To find out more, visit the [Solana server program page](https://solana.org/server-program).
 
 ## Stake Pools
 
-The Solana ecosystem operates various stake pools.  These stake pools delegate stake to validators using various delegation strategies. To learn more about stake pools and their delegation strategies, visit the [Solana Foundation stake pools page](https://solana.org/stake-pools).
\ No newline at end of file
+The Solana ecosystem operates various stake pools.  These stake pools delegate stake to validators using various delegation strategies. To learn more about stake pools and their delegation strategies, visit the [Solana Foundation stake pools page](https://solana.org/stake-pools).
```

Comment:
This chunk shouldn't be in the diff - I don't see any changes to the actual content so perhaps a line ending issue ?

Reply:
This item is not resolved - there are no changes to this text so this line should not be in the diff

Reply:
Yes, it is the problem with editor

Reply:
I fixed it, @steviez  check it.